extensions/ql-vscode/src/vscode-tests/jest.setup.ts

```diff
@@ -0,0 +1,9 @@
+import { env } from "vscode";
+
+(env as any).openExternal = () => {
+  /**/
+};
+
+afterAll(() => {
+  jest.restoreAllMocks();
```

Comment:
Any reason we're doing this here instead of defining `resetMocks` in the config? See https://jestjs.io/docs/configuration#resetmocks-boolean.

Reply:
I've now switched it to `resetMocks: true` and removed all calls to `mockReset` and `mockClear`.